py_trees/display.py

```diff
@@ -253,15 +257,34 @@ def get_node_attributes(node, visibility_level):
             attributes = (attributes[0], 'gray20', blackbox_font_colours[node.blackbox_level])
         return attributes
 
-    fontsize = 11
+    def get_node_label(node_name, behaviour):
+        """
+        This extracts a more detailed string (when applicable) to append to
+        that which will be used for the node name.
```

Comment:
Missing doc for parameters. Given that this is non public facing, I'm fine letting it pass.

Reply:
Aye